### ISSUE TITLE
Overload ostream operator for Pixel

### ISSUE THREAD

**Issue by simonspa** [simonspa/allpix-squared#29]
In gitlab by @simonspa on May 18, 2017, 13:36

It would be very useful to have the ostream operator overloaded for

```
using Pixel = ROOT::Math::DisplacementVector2D<ROOT::Math::Cartesian2D<int>>;
```

or, in general for all message objects so they can easily be printed.

**Comment by simonspa** [comment]
In gitlab by @Koensw on May 19, 2017, 21:12

changed milestone to %3

**Comment by simonspa** [comment]
In gitlab by @Koensw on Jun 22, 2017, 17:32

closed via commit 6ebaf99c334f818b9433b93fd780a1ae115dbf04
